### ISSUE TITLE
"Scroll Forward" not working under some conditions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#591]
```
What steps will reproduce the problem?
1. Portrait mode
2. Enable "Hide Status Bar" or "Hide System UI"
3. Assign "Scroll Forward" to any tap region

What is the expected output?
Tap the assigned region should go to page bottom or flip to next page

What do you see instead?
No response at all, regardless of the tap region assigned

What version of the product are you using?
2.0.6

On what operating system?


What device are you using?


Please provide any additional information below.

```

Original issue reported on code.google.com by `lumen.as...@gmail.com` on 20 Aug 2013 at 10:45

**Comment by GoogleCodeExporter** [comment]
```
OS/Device: Android 4.1.2 on Galaxy Note 2 
```

Original comment by `lumen.as...@gmail.com` on 20 Aug 2013 at 10:50

**Comment by GoogleCodeExporter** [comment]
```
Cannot reproduce. Can you repeat it on 2.0.5?
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 21 Aug 2013 at 9:27
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
Yes it's been like this since many revisions ago.

After some more testing, here are some additional steps to reproduce:

1. Remove Data via app manager (or reinstall EBookDroid)
2. Open a pdf file
3. Portrait mode, CLEAR all View menu settings EXCEPT Single Page, Hide System 
UI, Hide Status Bar
4. Set Zoom to 1.01x~1.17x
5. Try tapping the bottom to trigger Scroll Forward (won't  work).  Panning 
also wont work.  Scroll Back is unaffected, though. 

```

Original comment by `lumen.as...@gmail.com` on 21 Aug 2013 at 10:00

**Comment by GoogleCodeExporter** [comment]
```
Thank you for detailed description
```

Original comment by `Andrei.K...@gmail.com` on 22 Aug 2013 at 10:57

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.1
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 1:21
- Changed state: **Fixed**
